rust/regorule-library/src/userinfo/v1.rego

```diff
@@ -0,0 +1,23 @@
+package stackable.opa.userinfo.v1
+
+# Lookup by (human-readable) username
+userInfoByUsername(username) := http.send({
+  "method": "POST",
+  "url": "http://127.0.0.1:9476/user",
+  "body": {"username": username},
+  "headers": {"Content-Type": "application/json"},
+  "raise_error": true
```

Comment:
Just wanted to mention that we other examples where we assert on the return code and don't use `raise_error`:
https://github.com/stackabletech/demos/blob/61601f8ed879c4e8ba0766ec0fe0320ee959bdb7/stacks/end-to-end-security/trino-policies.yaml#L176-L184

I don't have any preference, but other ones might have and we should agree on one.

Reply:
I'm not seeing any discussion about it in https://github.com/stackabletech/demos/pull/22, at least. Is this a blocker, or can we come back to it later if it becomes an issue?

Reply:
I don't know enough about OPA. I'll be fine anyway I guess :)